borg/paperkey.html

```diff
@@ -0,0 +1 @@
+../docs/paperkey.html
```

Comment:
I did try without, but i didn’t have any luck. But i really don‘t grok setuptools/distutils. If you have any idea how to make this work so that the file can be shared between docs, `pip install -e .` and a real install (pyinstaller, pip install) i’d love to have something cleaner.